packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -853,6 +882,24 @@ export class Project extends ProjectBase {
     }
   }
 
+  public enableBatchBuilds(): IEnableBatchBuildsReturn | undefined {
+    if (!this._batchServiceRole) {
+      this._batchServiceRole = new iam.Role(this, 'BatchServiceRole', {
+        roleName: PhysicalName.GENERATE_IF_NEEDED,
+        assumedBy: new iam.ServicePrincipal('codebuild.amazonaws.com'),
+      });
+      this._batchServiceRole.addToPrincipalPolicy(new iam.PolicyStatement({
+        resources: [Lazy.string({
+          produce: () => this.projectArn,
+        })],
+        actions: ['codebuild:StartBuild', 'codebuild:StopBuild', 'codebuild:RetryBuild'],
```

Comment:
Can you format these on separate lines, so they're clearly visible?

```suggestion
        actions: [
          'codebuild:StartBuild',
          'codebuild:StopBuild',
          'codebuild:RetryBuild',
        ],
```